### PR TITLE
[PTX-23856] Increase Timeout In ValidateStorageCluster

### PR DESCRIPTION
**Pull request by kphalgun-px**
**What this PR does / why we need it**:
This PR sets the timeout for ValidateStorageCluster based on the number of storage driver nodes.

**Which issue(s) this PR fixes** (optional)
Closes #PTX-23856

**Special notes for your reviewer**:
There is no easy way to test this, so I will monitor the pipeline once the changes are merged.

### CODE REVIEW

**Review by spai-px — APPROVED** [review]
LGTM

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -5232,7 +5232,8 @@ func (d *portworx) ValidateDriver(endpointVersion string, autoUpdateComponents b
 		}
 
 		// Validate StorageCluster
-		if err = optest.ValidateStorageCluster(imageList, newStc, validateStorageClusterTimeout, defaultRetryInterval, true); err != nil {
+		storageClusterValidateTimeout := time.Duration(len(node.GetStorageDriverNodes())*9) * time.Minute
```
Comment (jainsmit) [inline drivers/volume/portworx/portworx.go:5235]: Why 9 minutes or 10 minutes for each nodes? How did you reach on that calculation ?

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Review by nikolaypopov — APPROVED** [review]

**Review by jainsmit — APPROVED** [review]
LGTM !!

File: drivers/volume/portworx/portworx.go [hunk 2]
```diff
@@ -5276,8 +5277,8 @@ func (d *portworx) updateAndValidateStorageCluster(cluster *v1.StorageCluster, f
 			}
 		}
 	}
-
-	if err = optest.ValidateStorageCluster(imageList, stc, validateStorageClusterTimeout, defaultRetryInterval, true); err != nil {
+	storageClusterValidateTimeout := time.Duration(len(node.GetStorageDriverNodes())*9) * time.Minute
```
Comment (sayalasomayajula-px) [inline drivers/volume/portworx/portworx.go:5280]: nit: i think it is repeated right , already defined at line 5235

**Review by sayalasomayajula-px — APPROVED** [review]

**Pull request merged**